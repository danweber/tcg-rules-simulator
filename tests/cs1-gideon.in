VERSION:2
# [All Turns] When an attack target is switched, by suspending this Tamer, \uff1cDraw 2\uff1e.",
# There's only one blocker? this effect isn't very good.
FIELD1:Gideon,CS1-Gideon Blocker,CS1-Obsidian
FIELD2:Attack1,DW1-Van
DECK1:CS1-02 CS1-03 CS1-04
GAME:P2 T3 MAIN 8
DUMP:HAND,DECK

ECHO: -- P2 attacks, block happens
CMD:ATTACK Attack1 Player
STEP:TO:1
STEP:AUTOANSWER:CAPTURE
DUMP:HAND,DECK
