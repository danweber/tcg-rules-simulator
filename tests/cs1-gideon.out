FIELD1:Gideon,CS1-Gideontheguardian Blocker,CS1-Obsidianshield
FIELD2:Attack1,DW1-Vanillamon
HAND1:
HAND2:
DECK1:CS1-Papershield CS1-Garbagecanlid CS1-Stoneshield
DECK2:
GAME:P2 T3 MAIN 8
ECHO: -- P2 attacks, block happens
CAPTURE:Activate Gideon The Guardian suspending
CAPTURE:Don't pay/activate
FIELD1:Gideon,CS1-Gideontheguardian,REST Blocker,CS1-Obsidianshield,REST
FIELD2:
HAND1:CS1-Garbagecanlid CS1-Stoneshield
HAND2:
DECK1:CS1-Papershield
DECK2:
GAME:P2 T3 MAIN 8
